ui/raidboss/data/04-sb/dungeon/doma_castle.txt

```diff
@@ -0,0 +1,137 @@
+hideall "--Reset--"
+hideall "--sync--"
+
+0.0 "--Reset--" sync / 00:0839:.*is no longer sealed/ window 5000 jump 0
+
+### Magitek Rearguard 
+# -ii 209F 20A0
+0 "Start" sync / 00:0839:The Third Armory will be sealed off/
+7.4 "Cermet Pile" sync /:Magitek Rearguard:209D:/
+15.0 "Garlean Fire" sync /:Magitek Rearguard:209E:/
+30.4 "Magitek Ray" sync /:Rearguard Bit:20A1:/
+35.4 "Mines" sync /03:........:Added new combatant Rearguard Mine/
+37.6 "Cermet Pile" sync /:Magitek Rearguard:209D:/
+40.6 "Magitek Ray" sync /:Rearguard Bit:20A1:/
+43.2 "Garlean Fire" sync /:Magitek Rearguard:209E:/
+48.7 "Magitek Ray" sync /:Rearguard Bit:20A1:/
+53.7 "Magitek Ray" sync /:Rearguard Bit:20A1:/
+60.7 "Cermet Pile" sync /:Magitek Rearguard:209D:/
+
+68.3 "Mines" sync /03:........:Added new combatant Rearguard Mine/
+70.5 "Cermet Pile" sync /:Magitek Rearguard:209D:/
+73.5 "Magitek Ray" sync /:Rearguard Bit:20A1:/
+76.2 "Garlean Fire" sync /:Magitek Rearguard:209E:/
+81.7 "Magitek Ray" sync /:Rearguard Bit:20A1:/
+85.8 "Cermet Pile" sync /:Magitek Rearguard:209D:/
+88.7 "Magitek Ray" sync /:Rearguard Bit:20A1:/
+91.4 "Garlean Fire" sync /:Magitek Rearguard:209E:/
+97.0 "Magitek Ray" sync /:Rearguard Bit:20A1:/
+98.5 "Mines" sync /03:........:Added new combatant Rearguard Mine/
+101.1 "Cermet Pile" sync /:Magitek Rearguard:209D:/
+104.1 "Magitek Ray" sync /:Rearguard Bit:20A1:/
+106.8 "Garlean Fire" sync /:Magitek Rearguard:209E:/
+112.3 "Magitek Ray" sync /:Rearguard Bit:20A1:/
+115.4 "Cermet Pile" sync /:Magitek Rearguard:209D:/
+118.4 "Magitek Ray" sync /:Rearguard Bit:20A1:/
+
+131.1 "Mines" sync /03:........:Added new combatant Rearguard Mine/ jump 68.3
+132.0 "Cermet Pile"
+135.0 "Magitek Ray"
+137.6 "Garlean Fire"
+143.1 "Magitek Ray"
+147.1 "Cermet Pile"
+150.1 "Magitek Ray"
+152.7 "Garlean Fire"
+158.2 "Magitek Ray"
+160.1 "Mines"
+
+### Magitek Hexadrone
+# -ii 20A4 20A6 20A7 2447
+1000 "Start" sync /00:0839:The Training Grounds will be sealed off/ window 1000,0
+1010.6 "Circle Of Death" sync /:Magitek Hexadrone:20A2:/
+1018.9 "2-Tonze Magitek Missile" sync /:Magitek Hexadrone:20A3:/
+1024.1 "Hexadrone Bits" sync /03:........:Added new combatant Hexadrone Bit/
+1029.6 "Bits Activate"
```

Comment:
Is there an ability name for the bits activating or the damage you take? It's fine if no, mostly just curious.  :)

Reply:
There isn't. Activating the tripwire has an ID, but that doesn't actually log until a second or so after triggering it. This timestamp is just my best guess at when the hitbox becomes active.